bin/PerfTest.cpp

```diff
@@ -75,7 +75,7 @@ int main(int argc, char *argv[]) {
         RooAbsData *dobs = w->data("data_obs");
         const RooCmdArg &constrainCmdArg = RooFit::Constrain(*mc_s->GetNuisanceParameters());
         std::unique_ptr<RooAbsReal> nll;
-        nll.reset(pdf.createNLL(*dobs, constrainCmdArg, RooFit::Extended(pdf.canBeExtended()), RooFit::Offset(true))); // make a new nll
+        nll.reset(pdf.createNLL(*dobs, constrainCmdArg, RooFit::Offset(true))); // make a new nll
```

Comment:
From the outset, this seems to change behavior? Does it not? Can you briefly explain why it is neede?

Reply:
The default behaviour of `createNLL` is the to create the extended likelihood if the pdf supports it. So this `RooFit::Extended(pdf.canBeExtended())` is redundant